README.md

```diff
@@ -1,51 +1,163 @@
+  * `SAUCE_USER_NAME` and `SAUCE_API_KEY`: [Sauce Labs][sauce] credentials for grunt test tasks (not used by the service itself)
+  * `ENABLE_ACCESS_LOG`: Any truthy value will enable writing an HTTP access log from Node. Useful if we are not running node behind a routing layer like nginx or heroku.
 
-## Installing as a service
 
-1. Install [git](http://git-scm.com/downloads) and [Node](http://nodejs.org) on your system
-2. Clone this repository to your system (`git clone git@github.com:Financial-Times/polyfill-service.git`)
-3. Run `npm install` (this will also run `grunt build` which you will need to do again if you edit any polyfills)
+Testing
+-------
 
-To run the app for **development**:
+The tests are split into for the service and tests for the polyfills. The polyfill tests require `SAUCE_USER_NAME` and `SAUCE_API_KEY` to be configured, view the [configuration](#configuration) section for more information.
 
-Run `grunt dev` from the root of the working tree.  This will watch your filesystem and automatically rebuild sources and restart if you make any changes to any of the app source code or polyfills.  If you change the docs, library or service (`docs`, `lib` and `service` directories) the service will be restarted.  If you change the polyfills or polyfill configs (`polyfills` directory), the polyfill sources will be recompiled before the service restarts, which makes the restart slightly slower.
+```sh
+grunt test           # run service tests and polyfill tests on a small set of browsers
+grunt simplemocha    # run the service tests
+grunt ci             # run the service tests and polyfills tests on a large set of browsers
+```
+
+We run the tests on CI, we can see the [results on CircleCI][ci].  `grunt ci` must pass before we merge a pull request.
+
+Release Process
+---------------
+
+ 1.  The release candidate is tested with the full grunt compatgen task to generate an updated compatibility table.
+ 2. The commit is tagged vX.Y.Z-rcN where N is initially 1
+ 3. Deploy to [QA](http://qa.polyfill.io)
+ 4. Announce the release on twitter
+ 5. Wait some number of days for feedback (usually 7 days). If necessary, make fixes, increment N, and return to step 1
+ 6. Update package.json version and github tag with vX.Y.Z
+ 7. Publish to npm
+ 8. Push to cdn.polyfill.io
+
+
+Deployment
+----------
+
+The [production][heroku-production] and [QA][heroku-qa] applications run on [Heroku].  The library is published to the public [npm] registry.
+
+Before creating a new deployment, update the compatability table:
+```sh
+grunt compatgen && git commit docs/assets/compat.json -m 'update compat.json'
+```
+
+We use [Semantic Versioning][semver] to tag releases.  Only tagged releases should hit production, this ensures that the `__about` endpoint is informative.  To tag a new release, use one of the following (this is the only time we allow a commit directly to `master`):
```

Comment:
"Only tagged releases should hit production, *which* ensures".

Also `/__about`